ecl/hqlcpp/hqltcppc2.cpp

```diff
@@ -651,8 +651,7 @@ IHqlExpression * CChildLinkedDatasetColumnInfo::buildSizeOfUnbound(HqlCppTransla
 
 void CChildLinkedDatasetColumnInfo::buildDeserialize(HqlCppTranslator & translator, BuildCtx & ctx, IReferenceSelector * selector, IHqlExpression * helper, IAtom * serializeFormat)
 {
-    if (isConditional())
-        checkAssignOk(translator, ctx, selector, queryZero(), sizeof(size32_t) + sizeof(byte * *));
```

Comment:
Note, this line previously did nothing since the first test is:

    if (matchesConstantValue(newSize, 0))
        return;

When I corrected it I noticed that what it was trying to check was unnecessary because it had already been done.